src/styles/_buttons.scss

```diff
@@ -0,0 +1,27 @@
+.icons {
+  @include button-style();
+  font-family: $icon-font;
+  font-size: 20px;
+}
+
+.icons:hover {
+  color: green;
+}
+
+.tooltipleft {
+  @include tooltip();
+  left: 1%;
+}
+
+.tooltipright {
```

Comment:
I like that you added these to the hover states. The right one is looking a little funky on my computer right now but I'm sure that will be ironed out once we finalize all the styling.